src/Network/Network/NetworkManager/DeployAzNetworkManagerCommit.cs

```diff
@@ -72,7 +72,7 @@ public class PostAzNetworkManagerCommitCommand : NetworkManagerBaseCmdlet
         [ResourceGroupCompleter]
         [ValidateNotNullOrEmpty]
         [SupportsWildcards]
```

Comment:
please update the help doc of the cmdlet

Reply:
Updated the doc to indicate Routing Security User as new scope accesses added.